battleship.py

```diff
@@ -10,40 +10,40 @@
 # sounds
 
 
-def hit_music():
+def hit_music():  # Plays serious music
     pygame.mixer.music.load('hardcore.mp3')
     pygame.mixer.music.play()
 
 
-def miss_music():
+def miss_music():  # Plays goofy music
     pygame.mixer.music.load('music.mp3')
     pygame.mixer.music.play()
 
 
-def skip_music():
+def skip_music():  # Plays recording of SammyOriginalSonicFan
     pygame.mixer.music.load('fricks.mp3')
     pygame.mixer.music.play(1)
 
 
-def hit_sound():
+def hit_sound():  # Plays a missile and explosion sound
     sound = pygame.mixer.Sound('388528__eflexmusic__artillery-explosion-close-mixed.wav')
```

Comment:
Don't forget to change the filenames of these sounds so it's not taking up so much space in the code.